build.sbt

```diff
@@ -86,7 +86,7 @@ libraryDependencies ++= Seq(
   ws,
   "uk.gov.hmrc"               %% "bootstrap-frontend-play-26"     % "2.24.0",
   "uk.gov.hmrc"               %% "domain"                         % "5.9.0-play-26",
-  "uk.gov.hmrc"               %% "govuk-template"                 % "5.55.0-play-26",
+  "uk.gov.hmrc"               %% "govuk-template"                 % "5.63.0-play-26",
   "uk.gov.hmrc"               %% "play-ui"                        % "8.15.0-play-26",
```

Comment:
Please remove the below snippet from applicaion.conf

`  google-analytics {
    token = UA-43414424-32
    host = auto
  }`

Reply:
My mistake, thanks for the heads up.